subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolutionResultApiIntegrationTest.groovy

```diff
@@ -324,8 +324,9 @@ baz:1.0 requested
                            println "\${it.cause} : \${it.description}"
                         }
                     }
-                    println 'Waiting 10s for the cache to expire'
-                    Thread.sleep(10100) // must be > 10s
+                    println 'Waiting for the cache to expire'
+                    // see org.gradle.api.internal.artifacts.ivyservice.resolveengine.store.CachedStoreFactory
+                    Thread.sleep(500) // must be > cache expiry
```

Comment:
⭕️ This may be a little tight depending on the timing of the expiry thread.

Reply:
Yeah, I changed it to 800ms.